src/Grpc.Net.ClientFactory/GrpcClientFactory.cs

```diff
@@ -32,6 +32,6 @@ public abstract class GrpcClientFactory
         /// <typeparam name="TClient">The gRPC client type.</typeparam>
         /// <param name="name">The configuration name.</param>
         /// <returns>A gRPC client instance.</returns>
-        public abstract TClient CreateClient<TClient>(string name) where TClient : ClientBase;
+        public abstract TClient CreateClient<TClient>(string name) where TClient : LiteClientBase;
```

Comment:
Should remove these constraints as well.

Reply:
Yes. Also add a unit test that uses client factory with Grpc.Core.Reflection package.